src/main/resources/org/jenkinsci/plugins/workflow/cps/replay/ReplayAction/index.jelly

```diff
@@ -4,27 +4,38 @@
     <l:layout title="Replay ${it.owner.displayName}" norefresh="true">
         <st:include page="sidepanel.jelly" it="${it.owner}"/>
         <l:main-panel>
-            <j:if test="${it.enabled}">
+            <j:if test="${it.enabled or it.rebuildEnabled}">
                 <h1>Replay ${it.owner.displayName}</h1>
                 <p>
                     <j:out value="${%blurb}"/>
                 </p>
-                <f:form action="run" method="POST" name="config">
```

Comment:
[diff for reviewers](https://github.com/jenkinsci/workflow-cps-plugin/pull/187/files?w=1#diff-bea9feb20818b01fa16e0538ec82cccfR12)